composer/core/precision.py

```diff
@@ -35,6 +35,12 @@ class Precision(StringEnum):
 def get_precision_context(precision: Union[str, Precision]) -> Generator[None, None, None]:
     """Returns a context manager to automatically cast to a specific precision.
 
+    .. warning::
+
+        :attr:`.Precision.FP16` is only supported when using DeepSpeed, as PyTorch does not
+        natively support this precision. When this function is invoked with :attr:`.Precision.FP16`,
+        this function will be a no-op.
```

Comment:
There's a difference between `get_precision_context` and the precision context itself.
```suggestion
        the precision context will be a no-op.
```